pytorch_lightning/trainer/connectors/optimizer_connector.py

```diff
@@ -29,11 +29,17 @@ def on_trainer_init(self) -> None:
         self.trainer.optimizers = []
         self.trainer.optimizer_frequencies = []
 
-    def update_learning_rates(self, interval: str, opt_indices: Optional[List[int]] = None) -> None:
+    def update_learning_rates(
+        self, interval: str, update_plateau_schedulers: bool, opt_indices: Optional[List[int]] = None
```

Comment:
lest move the `update_plateau_schedulers` to the end ad make default value which kind of actual behaviour

Reply:
I requested that is changed to required as we want people to knowingly choose which schedulers to update.